doc/api/dns.md

```diff
@@ -116,6 +116,28 @@ added: v8.3.0
 Cancel all outstanding DNS queries made by this resolver. The corresponding
 callbacks will be called with an error with code `ECANCELLED`.
 
+### `resolver.setLocalAddress([ipv4][, ipv6])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `ipv4` {string} A string representation of an IPv4 address.
+  **Default:** `'0.0.0.0'`
+* `ipv6` {string} A string representation of an IPv6 address.
+  **Default:** `'::0'`
+
+The resolver instance will send its requests from the specified IP address.
+This allows programs to specify outbound interfaces when used on multi-homed
+systems.
+
+If a v4 or v6 address is not specified, it is set to `0.0.0.0` or `::0`
```

Comment:
I don't have a great suggestion, but this paragraph is a bit awkward IMO.

Reply:
I would omit the bit about the adresses and just say it's chosen automatically by the operating system (IIRC that's what is happening unless c-ares does it differently).

Reply:
Tweaked the language in 61da1dc04b03d9ef5e06474a08100dc3373f7f5b

Reply:
Thanks, looking better.